tezos/new_context/src/tezedge_context.rs

```diff
@@ -717,12 +769,28 @@ impl ShellContextApi for TezedgeContext {
         Ok(MerkleStoragePerfReport::default())
     }
 
-    fn get_memory_usage(&self) -> Result<usize, ContextError> {
-        // TODO: Implement this
-        Ok(0)
+    fn get_memory_usage(&self) -> Result<ContextMemoryUsage, ContextError> {
+        let repository = self.index.repository.read()?;
+        let storage = (&*self.index.storage).borrow();
+
+        let usage = ContextMemoryUsage {
+            repo: repository.memory_usage(),
+            storage: storage.memory_usage(),
+        };
+
+        println!("usage={:?}", usage);
```

Comment:
Remove these prints that were used for collecting data on the storage deploy please.

Reply:
Btw, I am thinking that it may be useful to just log this data regularly (but using `slog` like the rest of the application) like we do for other things.

Reply:
Using `slog` would be great, right now every logs from the working tree is using `println`